superset-frontend/.eslintrc.js

```diff
@@ -17,7 +17,12 @@
  * under the License.
  */
 module.exports = {
-  extends: ['airbnb', 'prettier'],
+  extends: [
+    'airbnb',
+    'prettier',
+    'prettier/@typescript-eslint',
```

Comment:
why does the default need to extend prettier/@typescript? shouldn't this only be needed for `.ts` and `.tsx` files?

Reply:
Yea, it wasn't necessary. Fixed 🙂